llarp/service/auth.cpp

```diff
@@ -0,0 +1,20 @@
+#include "auth.hpp"
+#include <unordered_map>
+
+namespace llarp::service
+{
+  /// maybe get auth result from string
+  std::optional<AuthResult>
+  ParseAuthResult(std::string data)
+  {
+    static thread_local std::unordered_map<std::string, AuthResult> values = {
+        {"OKAY", AuthResult::eAuthAccepted},
+        {"REJECT", AuthResult::eAuthRejected},
+        {"PAYME", AuthResult::eAuthPaymentRequired},
+        {"LIMITED", AuthResult::eAuthRateLimit}};
```

Comment:
Why is this thread_local?  It looks like it could simply be a static initialized global const variable.